### ISSUE TITLE
Two Bugs on the demo home page

### ISSUE THREAD

**Issue by mgifford** [squidfunk/mkdocs-material#6837]
### Context

Microsoft's Accessibibility Insights (axe) reports two errors on the page. 

https://squidfunk.github.io/mkdocs-material/

### Bug description

Title: WCAG 1.4.1: Ensure links are distinguished from surrounding text in a way that does not rely on color (a[rel="noopener"][target="_blank"])
Tags: Accessibility, WCAG 1.4.1, link-in-text-block

Issue: Ensure links are distinguished from surrounding text in a way that does not rely on color (link-in-text-block - https://dequeuniversity.com/rules/axe/4.7/link-in-text-block?application=msftAI)

Target application: Material for MkDocs - https://squidfunk.github.io/mkdocs-material/

Element path: .md-copyright > a[rel="noopener"][target="_blank"]

Snippet: <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener"> Material for MkDocs Insiders </a>

Related paths: 
  .md-copyright

How to fix: 
Fix any of the following:
  The link has insufficient color contrast of 2.07:1 with the surrounding text. (Minimum contrast is 3:1, link text: #b8b8b9, surrounding text: #7c7d7f)
  The link has no styling (such as underline) to distinguish it from the surrounding text

Environment: Microsoft Edge version 122.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.41.0 (axe-core 4.7.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

Title: WCAG 2.1.1: Ensure elements that have scrollable content are accessible by keyboard (.md-search__scrollwrap)
Tags: Accessibility, WCAG 2.1.1, scrollable-region-focusable

Issue: Ensure elements that have scrollable content are accessible by keyboard (scrollable-region-focusable - https://accessibilityinsights.io/info-examples/web/scrollable-region-focusable)

Target application: Material for MkDocs - https://squidfunk.github.io/mkdocs-material/

Element path: .md-search__scrollwrap

Snippet: <div class="md-search__scrollwrap"> <div class="md-search-result" data-md-component="search-result"> <div class="md-search-result__meta">Type to start searching</div> <ol class="md-search-result__list" role="presentation"></ol> </div> </div>

How to fix: 
Fix any of the following:
  Element should have focusable content
  Element should be focusable

Environment: Microsoft Edge version 122.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.41.0 (axe-core 4.7.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

### Related links

- https://wave.webaim.org/
- https://squidfunk.github.io/mkdocs-material/

### Reproduction

Run https://accessibilityinsights.io/ on the page

### Steps to reproduce

1. Go to https://squidfunk.github.io/mkdocs-material/
2. After installing Accessibility Insights  https://accessibilityinsights.io (or just using WAVE) run it on the URL above.

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material/), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

**Comment by squidfunk** [comment]
Thanks for reporting. So, first of all, our landing page is an exception, because it is a custom template and does not belong to the standard templates that we are shipping with Material for MkDocs. Second of all, apart from the noise making it very hard to read (please write a summary next time), I understand that the following are the issues you want to report:

- The generator link "Material for MkDocs" in the footer is not underlined. A default underline of links in the body was already reported and heavily discussed in https://github.com/squidfunk/mkdocs-material/discussions/6238, which can be extended to the footer conceptually. We're currently not considering changing it, because it is a pretty fundamental change that might trip up many users, making more work for us as they create issues how to revert it.

  __Should we underline links and make them better discoverable? I think so, yes.__ But we cannot "just change" things with a project of this size. We will be revisiting the visual appearance of Material for MkDocs in the coming months and do a rebrush, trying to make all automated accessibility tools happy, but for the time being, please just add [additional CSS](https://squidfunk.github.io/mkdocs-material/customization/#additional-css) to style it to your taste. This will then be released as part of a new major release, denoting the breaking changes.

- The other issue I don't understand. It is very likely to be a false positive, because the search is perfectly usable with keyboard navigation and focusable. Try it out on our documentation, search for something, navigate with the keyboard. If you find something that does not work, please explain what does not work.

Automated accessibility tools are a great way of getting a glimpse of potential problems of a site when it comes to making it available to all audiences. However, in my experience, they often contain false positives, which need to be evaluated and assessed carefully.

I'm closing this issue as non-actionable for us. If you find that my explanations in the linked issue are not enough, or you find the search not accessible via keyboard, please provide [a detailed bug report that adheres to our guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).

**Comment by squidfunk** [comment]
Even one of your contributors commented that __the keyboard accessibility issue is a false positive__:
> https://github.com/CivicActions/guidebook/issues/1307#issuecomment-1783244756
